CODEOWNERS

```diff
@@ -3,17 +3,41 @@
 
 # Fallback to core group for crates or folders without specific owners, like docs/.
 
-* @unicode-org/icu4x-sc
+* @unicode-org/icu4x-owners
 
 # List of components with at least two owners as specified in docs/crate-ownership.
 
-components/datetime/            @zbraniecki @sffc
-components/locid/               @zbraniecki @nciric
-components/plurals/             @zbraniecki @sffc
-components/uniset/              @echeran
-ffi/ecma402/             @filmil
-provider/cldr/       @sffc
-provider/core/            @sffc
-provider/fs/         @sffc
-provider/testdata/             @sffc
-utils/fixed_decimal/            @sffc
+components/datetime/              @zbraniecki @gregtatum @nordzilla
+components/decimal/               @sffc
+components/icu/                   @unicode-org/icu4x-owners
+components/icu4x/                 @unicode-org/icu4x-owners
+components/locale_canonicalizer/  @dminor @zbraniecki
+components/locid/                 @zbraniecki @nciric
+components/plurals/               @zbraniecki @sffc
+components/uniset/                @echeran @iainireland
+experimental/bies/                @sffc
+experimental/calendar/            @Manishearth @sffc
+experimental/provider_ppucd/      @echeran
+experimental/segmenter/           @aethanyc @makotokato
```

Comment:
@sffc Could you invite @makotokato to join icu4x as he is added as the owner of the segmenter?